lib/ts/scratch.ts

```diff
@@ -0,0 +1,205 @@
+import * as jose from "jose";
```

Comment:
You might want to remove this or move it into the with-typescript folder (that's what we use to "test" typings in ci/scripts as well as a scratch file)